### PR TITLE
[DEPRECATED] feat: #30 Add banner for cookies usage

### PR DESCRIPTION
**Pull request by austininseoul**

### CODE REVIEW

File: frontend/lang/ca.js [hunk 1]
```diff
@@ -1,14 +1,16 @@
 export default {
   common: {
     loading: 'Carregant..',
-    language: 'Idioma',
-    calendar: 'Calendari'
```
Comment (jmformenti) [inline frontend/lang/ca.js:5]: The calendar label is still used

File: frontend/lang/ca.js [hunk 2]
```diff
@@ -251,8 +253,6 @@ export default {
     }
   },
   item: {
-    title: 'Títol',
```
Comment (jmformenti) [inline frontend/lang/ca.js:254]: These labels are still used

File: frontend/lang/en.js [hunk 3]
```diff
@@ -1,14 +1,16 @@
 export default {
   common: {
     loading: 'Loading..',
-    language: 'Language',
-    calendar: 'Calendar'
```
Comment (jmformenti) [inline frontend/lang/en.js:5]: Idem

File: frontend/lang/en.js [hunk 4]
```diff
@@ -251,8 +253,6 @@ export default {
     }
   },
   item: {
-    title: 'Title',
```
Comment (jmformenti) [inline frontend/lang/en.js:254]: Idem

File: frontend/lang/es.js [hunk 5]
```diff
@@ -1,14 +1,16 @@
 export default {
   common: {
     loading: 'Cargando..',
-    language: 'Idioma',
-    calendar: 'Calendario'
```
Comment (jmformenti) [inline frontend/lang/es.js:5]: Idem

File: frontend/lang/es.js [hunk 6]
```diff
@@ -251,8 +253,6 @@ export default {
     }
   },
   item: {
-    title: 'Título',
```
Comment (jmformenti) [inline frontend/lang/es.js:254]: Idem

File: frontend/lang/gl.js [hunk 7]
```diff
@@ -1,14 +1,16 @@
 export default {
   common: {
     loading: 'Cargando..',
-    language: 'Lingua',
-    calendar: 'Calendario'
```
Comment (jmformenti) [inline frontend/lang/gl.js:5]: Idem

File: frontend/lang/gl.js [hunk 8]
```diff
@@ -251,8 +253,6 @@ export default {
     }
   },
   item: {
-    title: 'Título',
```
Comment (jmformenti) [inline frontend/lang/gl.js:254]: Idem

File: frontend/lang/pt.js [hunk 9]
```diff
@@ -2,13 +2,15 @@ export default {
   menu: {
     common: {
       loading: 'Carregando..',
-      language: 'Linguagem',
-      calendar: 'Calendário'
```
Comment (jmformenti) [inline frontend/lang/pt.js:6]: Idem

File: frontend/lang/pt.js [hunk 10]
```diff
@@ -251,8 +253,6 @@ export default {
     }
   },
   item: {
-    title: 'Título',
```
Comment (jmformenti) [inline frontend/lang/pt.js:254]: Idem

File: frontend/layouts/default.vue [hunk 11]
```diff
@@ -144,6 +178,7 @@ export default {
     return {
       title_1: 'Philo',
       title_2: 'Biblon',
+      version: '0.6.0',
```
Comment (jmformenti) [inline frontend/layouts/default.vue:181]: Remove version from here. The version was moved to nuxt.config.js and is accessible via: $config.version

**Review by jmformenti — COMMENTED** [review]
I see some drawbacks:
* The current footer is being lost.
* There are active tags that are being removed.
* The privacy text should be in all languages.

File: frontend/lang/en.js (hunk 4 above)
Comment (austininseoul) [reply frontend/lang/en.js:254]: what does idem mean?

Comment (jmformenti) [reply frontend/lang/en.js:254]: It means "as above", in this case the label title is still used and can not be removed

**Comment by austininseoul** [comment]
https://www.loom.com/share/9fbddec568024e0fae15f9cd20eabb9a @jmformenti

**Comment by jmformenti** [comment]
@austininseoul It looks good, but the pull request needs to be updated.

**Pull request closed without merging**